src/tuf.rs

```diff
@@ -9,19 +9,19 @@ use crate::crypto::PublicKey;
 use crate::error::Error;
 use crate::interchange::DataInterchange;
 use crate::metadata::{
-    Delegation, Delegations, Metadata, MetadataPath, Role, RootMetadata, SignedMetadata,
-    SnapshotMetadata, TargetDescription, TargetsMetadata, TimestampMetadata, VirtualTargetPath,
+    Delegations, Metadata, MetadataPath, Role, RootMetadata, SignedMetadata, SnapshotMetadata,
+    TargetDescription, TargetsMetadata, TimestampMetadata, VirtualTargetPath,
 };
 use crate::Result;
 
 /// Contains trusted TUF metadata and can be used to verify other metadata and targets.
```

Comment:
it seems a bit redundant to have `trusted_` prefixes on the fields themselves since `Tuf` only ever stores trusted metadata.  I'm fine with either though.

Reply:
Agree it's quite redundant. However, since the `Tuf` type is the thing that manages the security properties of rust-tuf, I figured it couldn't hurt to have additional signal. That's also why I wrapped these types in a `Verified` struct so that we can get the compiler to enforce we're using the right thing.